### PR TITLE
Day 2 project Hello Microverse project 

### PR DESCRIPTION
**Pull request by gaks1**
What's done:
1.  Made and committed a .gitignore file.
2.  Made and committed all linters and necessary files.
3. Added and committed html, css and text.md files.
4. Updated Readme file according to code reviewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+## Copyright 2023, [Isaac Gakure Wanyoike]
+
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [APP TYPE].
+
+THE [APP TYPE] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [APP TYPE] OR THE USE OR OTHER DEALINGS IN THE [APP TYPE].# MicroverseProject1
```
Comment (jerryowusu) [inline README.md]: Your readme file contains only the license text. Kindly ensure that you update your readme file with the microverse [readme template](https://github.com/microverseinc/readme-template) and customized it according to your credentials to make it more professional and appealing. :+1:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @gaks1 , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

no-we-were-so-close-pauline-turner-brooks

## Highlights
All linters are passing :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Required Changes ♻️

- [ ] Your pull request title is not comprehensible and does not tell about the whole project. Kindly ensure that your pull request title is descriptive telling others what changes are inside in a comprehensible way. 
- [ ] Kindly ensure that your pull request have a summary describing introduced changes in more detail since there is none currently. :wink: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
-# MicroverseProject1
\ No newline at end of file
+## Copyright 2023, [Isaac Gakure Wanyoike]
+
+
+
+This is a work done for Microverse projects.
+
+DONE ON 10/01/2023
```
Comment (Gambit142) [inline README.md]: - [ ] As instructed by the previous reviewer, kindly make your README.md file look professional by using the recommended template provided by Microverse. Be sure to add the relevant headers that are necessary for this project. You can use the template link provided below to make yours:

https://github.com/microverseinc/readme-template

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @gaks1 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- Great work making the pull request descriptive and understandable ✔️ 
- All linters checks passed ✔️ 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
- For subsequent projects, it would be nice to make your commit messages in imperative tense and capitalized.

### Example of Good Commit Message

- **_Add_** a git ignore file

### Example of Bad Commit Message

- _**added**_ a git ignore file
- _**Added**_ a git ignore file


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (gaks1) [reply README.md]: Hello,  I cannot resubmit as it says it says the review is pending.

Comment (Gambit142) [reply README.md]: @gaks1, you just resubmitted and I just claimed it again, so is there a reason why you stated this? 😄😆 👍

Comment (gaks1) [reply README.md]: > @gaks1, you just resubmitted and I just claimed it again, so is there a reason why you stated this? 😄😆 👍

@Gambit142 , sorry I did not reload the progress page so I was confused and panicked.

**Review by Gambit142 — APPROVED** [review]
Hi @gaks1 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Pull request summary is descriptive ✔️ 

## Optional suggestions
- For subsequent projects, it would be nice to make your commit messages in imperative tense and capitalized. I highlighted this as an optional suggestion when I made my first review. It is important you stick to conventions and take notes of this when implementing other projects. 👍 

### Example of Good Commit Message

- **_Add_** a git ignore file
- Update trial readme

### Example of Bad Commit Message

- _**added**_ a git ignore file
- _**Added**_ a git ignore file
- trial 2 readme update

#### Kindly note that each of the good commit messages starts with an imperative tense and the first letter is capitalized. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by gaks1** [comment]
@Gambit142 @jerryowusu , do we delete the branch

File: README.md (hunk 2 above)
Comment (Gambit142) [reply README.md]: @gaks1, it is alright. I understand. 👍

**Comment by Gambit142** [comment]
> @Gambit142 @jerryowusu , do we delete the branch

@gaks1, if you wish to. 👍

**Pull request merged**